nss-evening-cohort-13/star-destroyer-dashboard-star-destroyer-dashboard
# Milesstone 2: Create, Read, Update, and Delete for Personnel

## Description

Provides CRUD functionality for Personnel, also creates a foundation for clicking on pages and upgrades styles.

## Related Issue

[https://github.com/nss-evening-cohort-13/star-destroyer-dashboard-star-destroyer-dashboard/issues/5](https://github.com/nss-evening-cohort-13/star-destroyer-dashboard-star-destroyer-dashboard/issues/5)

## Motivation and Context

It solves our milestone of CRUD for personnel

## How Can This Be Tested?

git fetch origin development

Check the published URL - [https://star-destroyer-a0766.web.app/](https://star-destroyer-a0766.web.app/) published 4:50pm 10/15

## Screenshots (if appropriate):

## Types of changes



- [X ] New feature (non-breaking change which adds functionality)
